src/rt/sections_osx.d

```diff
@@ -23,6 +23,13 @@ import core.sys.osx.mach.getsect;
 import rt.deh, rt.minfo;
 import rt.util.container.array;
 
+version (D_LP64)
+    import rt.sections_osx64;
+else
+    import rt.sections_osx32;
+
+mixin SectionsOSX;
```

Comment:
Ok, but I strongly believe that any of the solutions that I've suggested will have us in a better spot than a mixin.